<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mqCameraControlsWidget</class>
 <widget class="QWidget" name="mqCameraControlsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>27</width>
    <height>186</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="CameraCentreOfMassToggle">
         <property name="toolTip">
          <string>Toggle camera : centered at (x,y,z) = (0,0,0) / or at the centre of mass of all opened objects</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../Icons/icons.qrc">
           <normaloff>:/Icons/move_cam.png</normaloff>
           <normalon>:/Icons/move_cam2.png</normalon>:/Icons/move_cam.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="CameraOrthoPerspectiveToggle">
         <property name="toolTip">
          <string>Toggle camera : orthographic or perspective projection</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Icons/icons.qrc">
           <normaloff>:/Icons/camera_ortho.png</normaloff>
           <normalon>:/Icons/camera_persp.png</normalon>:/Icons/camera_ortho.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="CameraFront">
         <property name="toolTip">
          <string>Move Camera: view objects from front side</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Icons/icons.qrc">
           <normaloff>:/Icons/camera_front.png</normaloff>:/Icons/camera_front.png</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="CameraBack">
         <property name="toolTip">
          <string>Move Camera: view objects from back side</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../Icons/icons.qrc">
           <normaloff>:/Icons/camera_back.png</normaloff>:/Icons/camera_back.png</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="CameraRight">
         <property name="toolTip">
          <string>Move Camera: view objects from right side</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../Icons/icons.qrc">
           <normaloff>:/Icons/camera_right.png</normaloff>:/Icons/camera_right.png</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="CameraLeft">
         <property name="toolTip">
          <string>Move Camera: view objects from left side</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../Icons/icons.qrc">
           <normaloff>:/Icons/camera_left.png</normaloff>:/Icons/camera_left.png</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="CameraAbove">
         <property name="toolTip">
          <string>Move Camera: view objects from above</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../Icons/icons.qrc">
           <normaloff>:/Icons/camera_above.png</normaloff>:/Icons/camera_above.png</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="CameraBelow">
         <property name="toolTip">
          <string>Move Camera: view objects from below</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../Icons/icons.qrc">
           <normaloff>:/Icons/camera_below.png</normaloff>:/Icons/camera_below.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
